engine/api/project/dao_platform.go

```diff
@@ -0,0 +1,160 @@
+package project
+
+import (
+	"database/sql"
+	"encoding/base64"
+	"encoding/json"
+
+	"github.com/go-gorp/gorp"
+
+	"github.com/ovh/cds/engine/api/platform"
+	"github.com/ovh/cds/engine/api/secret"
+	"github.com/ovh/cds/sdk"
+)
+
+// PostGet is a db hook
+func (pp *dbProjectPlatform) PostGet(db gorp.SqlExecutor) error {
+	model, err := platform.LoadModel(db, pp.PlatformModelID)
+	if err != nil {
+		return sdk.WrapError(err, "dbProjectPlatform.PostGet> Cannot load model")
+	}
+	pp.Model = model
+
+	query := "SELECT config FROM project_platform where id = $1"
+	s, err := db.SelectNullStr(query, pp.ID)
+	if err != nil {
+		return sdk.WrapError(err, "dbProjectPlatform.PostGet> Cannot get config")
+	}
+	if s.Valid {
```

Comment:
why not using `gorpmapping.JSONNullString`

Reply:
don't want to create a type for 1 field :/